test/fixtures/for_unit_tests/api_versions.txt

```diff
@@ -22,6 +22,9 @@ events.k8s.io/v1beta1
 extensions/v1beta1
 networking.k8s.io/v1
 networking.k8s.io/v1beta1
+networking.gke.io/v1
```

Comment:
I'm hesitant to add gke versions here. This isn't something everyone will be able replicate.

@timothysmith0609 thoughts?

Reply:
Since I didn't add a test for the pruning of ServiceNetworkEndpointGroup specifically I can just remove changes in the tests. It doesn't really add anything except make the expected count go from 25 to 26.

Reply:
>I'm hesitant to add gke versions here

I agree, I think omission is ok, here